test/cases/ami_empty_blueprint.json

```diff
@@ -130,7 +130,7 @@
       "options": {
         "format": "raw.xz",
         "filename": "image.raw.xz",
-        "size": 3222274048,
+        "size": 6442450944,
```

Comment:
Why are the sizes changing? Should be some argument why the new sizes are right wherever the change was introduced. Also, this needs to be squashed with the commit that introduce the change to make sure the tests keep passing.